build: Add libcurl dependency to libvirt_driver.la
author	Jiri Denemark <jdenemar@redhat.com>	
Fri, 1 Feb 2013 12:36:06 +0000 (13:36 +0100)
committer	Jiri Denemark <jdenemar@redhat.com>	
Tue, 5 Feb 2013 09:00:32 +0000 (10:00 +0100)
libvirt.c calls curl_global_init() if WITH_CURL is defined and thus it
should be linked with libcurl. This fixes link failure in case neither
xenapi nor esx driver is enabled (they are the only users of libcurl).

src/Makefile.am		patch | blob | history
diff --git a/src/Makefile.am b/src/Makefile.am
index 070a089..f6162df 100644 (file)
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -792,9 +792,11 @@ noinst_LTLIBRARIES += libvirt_driver.la
 libvirt_la_BUILT_LIBADD += libvirt_driver.la
 libvirt_driver_la_SOURCES = $(DRIVER_SOURCES)
 
-libvirt_driver_la_CFLAGS = $(NUMACTL_CFLAGS) $(GNUTLS_CFLAGS) \
+libvirt_driver_la_CFLAGS = \
+               $(NUMACTL_CFLAGS) $(GNUTLS_CFLAGS) $(CURL_CFLAGS) \
                -I$(top_srcdir)/src/conf $(AM_CFLAGS)
-libvirt_driver_la_LIBADD = $(NUMACTL_LIBS) $(GNUTLS_LIBS) $(DLOPEN_LIBS)
+libvirt_driver_la_LIBADD = \
+               $(NUMACTL_LIBS) $(GNUTLS_LIBS) $(CURL_LIBS) $(DLOPEN_LIBS)
 
 USED_SYM_FILES = $(srcdir)/libvirt_private.syms
